backend/models/Chat.js

```diff
@@ -2,17 +2,20 @@ const mongoose = require("mongoose");
 const { Schema } = mongoose;
 
 const ChatSchema = new Schema({
-  users: {
-    type: [{ type: Schema.Types.ObjectId, ref: "User" }],
-    validate: {
-      validator: function (users) {
-        return users.length > 0;
-      },
-      message: "A chat must have at least one user.",
-    },
-  },
-  message: [{ type: Schema.Types.ObjectId, ref: "Message" }],
-  polls: [{ type: Schema.Types.ObjectId, ref: "Poll" }],
+	name: {
+		type: String,
+		required: true,
+	},
+	pin: {
+		type: Number,
+		required: true,
+		unique: true,
+	},
+	users: {
```

Comment:
Why do you remove the validation part?

Reply:
I'll add the validation back in when we have some sort of guest user handling

Reply:
Why we need that now? A room must have an user even if it's just guest

Reply:
Because we dont have a user_id for the guest user if they don't exist, so it will just crash every time.

Reply:
I don't really get what you mean, every user we create will have an associate _id field with them in the database

Reply:
You can create a guest user using User.create({})